src/Symfony/Component/HttpKernel/DependencyInjection/AddCacheClearerPass.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * Registers the cache clearers.
+ *
+ * @author Dustin Dobervich <ddobervich@gmail.com>
+ */
+class AddCacheClearerPass implements CompilerPassInterface
```

Comment:
Hum, then this would need to be in the `DependencyInjection` component, but I am not sure this should be the goal of this PR (but a later one probably ?)